### PR TITLE
Changed duration

### LINKED ISSUE #19003: Default SnackBar duration is below Material Spec's minimum
The default duration in Flutter is 1.5s. https://docs.flutter.io/flutter/material/SnackBar/duration.html

However, the [Material spec](https://material.io/design/components/snackbars.html#behavior) says the **minimum** time it should be displayed is 4 seconds. 

> Behavior 
> 
> Appearing and disappearing
> Snackbars appear without warning, and don't require user interaction. They automatically disappear from the screen after a minimum of four seconds, and a maximum of...
> 
> READ MORE
> Snackbars appear without warning, and don't require user interaction. They automatically disappear from the screen after a minimum of four seconds, and a maximum of ten seconds.
>

### PR DESCRIPTION
**Pull request by jslavitz**
fixes https://github.com/flutter/flutter/issues/19003

### CODE REVIEW

File: packages/flutter_tools/lib/src/commands/create.dart [hunk 1]
```diff
@@ -307,17 +307,14 @@ To edit platform code in an IDE see https://flutter.io/developing-packages/#edit
     int generatedCount = 0;
     generatedCount += _renderTemplate('create', projectPath, templateContext);
     generatedCount += _injectGradleWrapper(projectPath);
-
```
Comment (jonahwilliams) [inline packages/flutter_tools/lib/src/commands/create.dart]: Nit: Undo unrelated change

**Review by jonahwilliams — CHANGES_REQUESTED** [review]
LGTM with minor nit above.  Even though the default duration is technically a breaking change I don't think it is severe enough to merit a breaking change announcement.

**Comment by Hixie** [comment]
fyi @HansMuller

**Comment by Hixie** [comment]
This needs a test.

**Review by jonahwilliams — APPROVED** [review]
LGTM

**Comment by jslavitz** [comment]
@Hixie - This one too?

**Comment by HansMuller** [comment]
The [spec](https://material.io/design/components/snackbars.html#behavior) say:

```
Snackbars appear without warning, and don't require user interaction. They automatically disappear from the screen after a minimum of four seconds, and a maximum of ten seconds.
```

So this change looks good. It might be worth passing along the spec's guidance about 4-10 seconds in the doc for `duration`.

File: packages/flutter/lib/src/material/snack_bar.dart [hunk 2]
```diff
@@ -169,7 +169,7 @@ class SnackBar extends StatelessWidget {
 
   /// The amount of time the snack bar should be displayed.
   ///
-  /// Defaults to 1.5s.
+  /// Defaults to 4.0s.
```
Comment (HansMuller) [inline packages/flutter/lib/src/material/snack_bar.dart]: The material spec link above this line isn't valid (anymore). It should be https://material.io/design/components/snackbars.html#behavior

Comment (jslavitz) [reply packages/flutter/lib/src/material/snack_bar.dart]: You want me to copy in that whole "Snackbars appear without warning, and don't require user interaction. They automatically disappear from the screen after a minimum of four seconds, and a maximum of ten seconds."?

Comment (HansMuller) [reply packages/flutter/lib/src/material/snack_bar.dart]: I was just suggesting that you replace the URL on line 179 (`https://material.google.com/components/snackbars-toasts.html`)

Comment (jslavitz) [reply packages/flutter/lib/src/material/snack_bar.dart]: Just changed that!

**Review by HansMuller — APPROVED** [review]
LGTM. Sadly the snackbar link needs another update.
